### DISCUSSION TITLE
style.borders() method doesn't work

### DISCUSSION THREAD

**Discussion by wanderer1037** [posit-dev/great-tables#378] [General]
Hi, there!
style.borders() method doesn't work! Anyone else having the same issue?
```
from great_tables import GT, from_column, style, loc
from great_tables.data import airquality
air_head = airquality.head()
gt_air = GT(air_head)
gt_air.tab_style(
    style=style.borders(sides="left", color="#74a9cf", style="dashed", weight="3px"),
    locations=loc.body(rows=[1, 2])
)
```
![image](https://github.com/posit-dev/great-tables/assets/165550726/b579c18c-3c8c-4d88-8e2a-37d065cb5e0e)

**Comment by machow** [comment]
Is this in VS Code? Up until recently, VS Code had hard-coded styles that disabled borders. Not sure if the fix is released, but here is the issue:

https://github.com/microsoft/vscode/issues/210462

edit: I can see borders in jupyter lab. One option is to use the `.save()` method and see if there are borders in the image.

<img width="901" alt="image" src="https://github.com/posit-dev/great-tables/assets/2574498/7cdcc12e-e717-42c3-ae89-960120648db7">
